roles/qe-server/files/x11vnc@.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=x11vnc allows VNC connections to real X11 displays
+# TODO: it's not possible to reference parametrized requirements like that
+# Requires=Xvfb@%I.service
+# After=Xvfb@%I.service
+
+[Service]
+Type=forking
+ExecStart=/usr/bin/x11vnc -display %I -bg -nopw -xkb -forever -shared -logfile /var/log/x11vnc.%I.log
```

Comment:
As I mentioned above, would it be possible to specify also the port number determined from the display number?